datasets/adult/metadata.yaml

```
@@ -1,42 +1,171 @@
 #Generated automatically by pmlb/write_metadata.py
 dataset: adult
-description: None yet. See our contributing guide to help us add one.
-source: None yet. See our contributing guide to help us add one.
-publication: None yet. See our contributing guide to help us add one.
+description: Predictipn of if person make over $50K a year, based on 14 other features.
+source: >
+  http://mlr.cs.umass.edu/ml/datasets/Adult
+  https://www.openml.org/d/1590
+publication: >
+  Ron Kohavi, "Scaling Up the Accuracy of Naive-Bayes Classifiers: a Decision-Tree Hybrid", 
+  Proceedings of the Second International Conference on Knowledge Discovery and Data Mining, 1996
 task: classification
 target:
   type: binary
-  description: None yet. See our contributing guide to help us add one.
-  code: None yet. See our contributing guide to help us add one.
+  description: If a given adult makes over $50K a year
+  code: >
+    ' <=50K' = 0, 
```

```suggestion
    ' <=50K' = 1, 
```
I think the coding is actually reversed here.